### ISSUE TITLE
Tertiary color does not have correct Color Contrast ratio.

### ISSUE THREAD

**Issue by dhoehna** [microsoft/microsoft-ui-xaml#9624]
### Describe the bug

The tertiary color brush does not have the correct CC ratio.

### Steps to reproduce the bug

1. Open up the WinUI 3 Gallery
2. Go to Design Guidance -> Colors
3. Open accessibility Insights for Windows.
4. In the accessibility insights, go to the Color Contrast Analyzer.
5. Turn on Auto-detect contrast ratios.
6. Hover over the tertiary text in the WinUI 3 Gallery.

### Expected behavior

The CC ratio is over 4.0

### Screenshots

![image](https://github.com/microsoft/microsoft-ui-xaml/assets/2517139/458a9bb5-2a35-41f5-8afa-798d7903c23c)

### NuGet package version

WinUI 3 - Windows App SDK 1.5.3: 1.5.240428000

### Windows version

Windows Insider Build (xxxxx)

**Comment by niels9001** [comment]
This is by design. The color guidance calls out specifically that these brushes are 'not accesible' and should only be used in Hover text states within the context of a button. The button's other visual properties would ensure contrast ratios are met.

<img width="680" height="222" alt="Image" src="https://github.com/user-attachments/assets/f88dc2a6-a6ee-4bfa-a00d-692f380a4eb2" />
